qadence/blocks/primitive.py

```diff
@@ -381,10 +381,6 @@ def _block_title(self) -> str:
     def __ascii__(self, console: Console) -> RenderableType:
         raise NotImplementedError
 
-    @property
-    def n_qubits(self) -> int:
```

Comment:
Is this redundant with the base class property ?

Reply:
Not just redundant, it was overriding the `max(qubit_support)+1` which would cause wrong results in `block_to_tensor`.